scripts/main.py

```diff
@@ -1,8 +1,37 @@
-import sys
-from image import parse_image
+from PIL import Image
+import numpy as np
 
-image = parse_image(sys.argv)
+def get_average_color(left, right, upper, lower):
+	r, g, b = 0, 0, 0
+	for x in range(left, right):
+		for y in range(upper, lower):
+			r += image.getpixel((x, y))[0]
+			g += image.getpixel((x, y))[1]
+			b += image.getpixel((x, y))[2]
+	pixel_count = chunk_width*chunk_height
+	r = int(r/pixel_count)
+	g = int(g/pixel_count)
+	b = int(b/pixel_count)
+	return (r, g, b)
 
-print(image)
+image = Image.open('images/image3.jpg')
 
-image.show()
\ No newline at end of file
+dimension = 3456
+
+chunk_width = int(image.width/dimension)
+chunk_height = int(image.height/dimension)
+
+chunk_averages = np.empty([dimension, dimension], dtype=object)
+
+for x in range(0, dimension):
+	for y in range(0, dimension):
+		left, right, upper, lower = x*chunk_width, x*chunk_width + chunk_width, y*chunk_height, y*chunk_height + chunk_height
```

Comment:
In my opinion this would be much clearer as x_1, y_1, x_2, y_2, with relevant method change:

```
def get_average_color(x_1, y_1, x_2, y_2):
	r, g, b = 0, 0, 0
	for x in range(x_1, x_2):
		for y in range(y_1, y_2):
			r += image.getpixel((x, y))[0]
			g += image.getpixel((x, y))[1]
			b += image.getpixel((x, y))[2]
	...
```